acceptance/tests/api-gateway_v2/main_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apigateway_v2
```

Comment:
minor but can we name this package `apigatewayv2` https://go.dev/doc/effective_go#package-names